recipes/stb/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "20200203":
     url: "https://github.com/nothings/stb/archive/0224a44a10564a214595797b4c88323f79a5f934.zip"
     sha256: "5dd9cf8a9a8c0f253fad3e9904923b2dfc740eddbc86415c3f76152bd534f23a"
+  "20210713":
```

Comment:
We have started to use a `cci.YYYYMMDD` format.
But I don't think `tools.Version` is compatible with that.

Reply:
True, it's incompatible.

Reply:
Should I also change it for version `20200203`?

Reply:
Please do. Older versions remain available on conan-center.
To tackle the version comparison, perhaps create a helper method `self._version` and use that for version comparison
```python
def _version(self):
    return str(self.version)[4:]
```

Alternatively, don't change the version and use:
```
def _version(self):
    import re
    return re.match(r"^(?:cci\.|)(.*)", str(self.version)).group(1)
```